### DISCUSSION TITLE
The creeper does not explode near the player

### DISCUSSION THREAD

**Discussion by 1mrasts** [FabricMC/fabric-api#2204] [Mod Dev Support]
I created my creeper by transferring all the code from CreeperEntity (I wrote "extends HostileEntity implements IAnimatable") The creeper has been created, everything is fine, but it does not explode when it approaches the player, it explodes only if you click on it with a lighter, what to do?

**Comment by warjort** [comment]
It is nearly impossible to help you if you are just going to assert that you have done things correctly.
You need to show the relevant code so we can see what you have actually done (rather than what you think you have done).
Then we might be able to find the error.

Guessing: Since you don't mention it, you haven't done your equivalent of the ````CreeperIgniteGoal```` in your AI or it is incorrect.
